internal/cli/option/flag.go

```diff
@@ -4,9 +4,9 @@ import "github.com/spf13/pflag"
 
 type FlagFn func(*pflag.FlagSet)
 
-func ServerURL(v *string) FlagFn {
+func Insecure(v *bool) FlagFn {
 	return func(fs *pflag.FlagSet) {
-		fs.StringVar(v, "server", "", "Server URL")
+		fs.BoolVarP(v, "insecure", "k", false, "Skip TLS certificate verification")
```

Comment:
1. I think the flag name is a bit generic. I regret that argo-cd also uses `--insecure` to mean insecure tls.

One practice that I've seen (at least with other golang apps) is to prefix all insecure options with `--insecure-` and then the thing they are making insecure. e.g. kubectl names it `--insecure-skip-tls-verify`. Then later we could introduce another `--insecure-xxx` flag should one arise.

2. I would also choose not to add the `-k` shorthand flag. I feel the fact that it is insecure should be in your face with an obnoxiously long `insecure-skip-tls-verify` flag, and don't want the `k` to occupy some other useful shorthand. I'm guessing the `-k` option was chosen following from `curl`, but I think the curl use case deals with a lot with insecure servers and so it makes sense for the shorthand to exist in curl.

Reply:
+1

Reply:
> I'm guessing the -k option was chosen following from curl

Yes... in fact, both `--insecure` and `-k` were chosen for that reason. I thought the similarity to curl would make them intuitive both to understand and to use, but I am happy to change this since I'm outnumbered on this. :smile:

Reply:
Fixed. Changed it to `--insecure-skip-tls-verify` as suggested and restored `k` to what is was originally -- shorthand for the `login` command's `--kubeconfig` option.